src/lib/markbind/src/lib/markdown-it-attributes/index.js

```diff
@@ -0,0 +1,21 @@
+const markdownIt = require('markdown-it')({
+  html: true,
+  linkify: true
+});
+
+markdownIt.use(require('markdown-it-mark'))
```

Comment:
I opted for maintaining parity with past behaviours for now for this, as the reasoning for the omission of some of the plugins are clear ( e.g. `markdown-it-task-lists` wouldn't be used in attributes ), while I'm not so sure about some other plugins.

Markdown-it plugins vue-strap has but MarkBind dosen't:
- markdown-it-sub
- markdown-it-sup

Markdown-it plugins MarkBind has but vue-strap dosen't:
- markdown-it-attrs
- markdown-it-footnotes
- markdown-it-linkify-images

Markdown-it plugins MarkBind has but vue-strap dosen't, but would not be used in attributes anyway:
- markdown-it-table-of-contents
- markdown-it-task-lists
- markdown-it-block-embed
- markdown-it-radio-button 

It might be good to reach a standardisation decision for this in a separate PR instead, in case it has some side effects